lamindb/models/history.py

```diff
@@ -0,0 +1,53 @@
+from django.db import models
+
+
+class History(models.Model):
+    """Stores the edit history for all LaminDB tables."""
+
+    seqno = models.AutoField(primary_key=True)
+    migration_history_id = models.BigIntegerField()
+    id = models.UUIDField()
+    table_name = models.CharField(max_length=100)
```

Comment:
If we need to track chars here let's be save and not limit at `max_length=100` but at least allow `255`. Because django concatenates app name and table name and table names can indeed get long that's an unnecessary risk.

Given we have an enumeration of tables I still think that having an id rather than a char name would be nice.

Reply:
Unfortunately `contenttypes`' enumeration of tables doesn't include many-to-many tables. I can create a separate table that tracks all table names to make sure that tables have unique IDs that are shared across instances if you think that'd be helpful.

Reply:
> I can create a separate table that tracks all table names to make sure that tables have unique IDs
I think the table of tables would be helpful, yes. 🤔